authentication/serializers.py

```diff
@@ -1,66 +1,81 @@
 from django.contrib.auth import authenticate, get_user_model
 from django.conf import settings
 from django.core.exceptions import ValidationError
+from rest_framework.validators import UniqueValidator
 from djoser.serializers import UserCreatePasswordRetypeSerializer, UserSerializer, TokenCreateSerializer
 from rest_framework import serializers
 
+from profiles.models import Profile
 from validation.validate_password import validate_password_long, validate_password_include_symbols
 
 
 User = get_user_model()
 
 
+class CustomProfileSerializer(serializers.ModelSerializer):
```

Comment:
So we have now two separate serializers for Profile? Can you explain a bit why?

Reply:
The CustomProfileSerializer is used as nested in UserRegistrationSerializer to handle the company fields, that is needed while user registration.